covid_moonshot_ml/datasets/utils.py

```diff
@@ -93,6 +45,23 @@ def add_seqres(pdb_in, pdb_out=None):
     print(f"Wrote {pdb_out}", flush=True)
 
 
+def seqres_to_res_list(SEQRES_str):
```

Comment:
```suggestion
def seqres_to_res_list(seqres_str):
```
I think Python variable names shouldn't have capital letters

Reply:
done here: https://github.com/choderalab/covid-moonshot-ml/pull/58/commits/4486d0fd4a101824941f03ff1cde7e636a3b5c15